src/Sylius/Bundle/CoreBundle/spec/Fixture/Factory/TaxonExampleFactorySpec.php

```diff
@@ -25,7 +25,7 @@
 
 class TaxonExampleFactorySpec extends ObjectBehavior
 {
-    function let(
+    public function let(
```

Comment:
All our specs don't have any `public` accessor. Can you revert this change?

Reply:
Done! It was the auto formatter from my editor 👀

Reply:
but there is one more file to fix :P https://github.com/Sylius/Sylius/pull/10952/files#diff-f275e213fd9b5b34a08b735a342f79d9R28

Reply:
😩 

Fixed.